### PR TITLE
data: project addition by Hardikaz #714

### PR DESCRIPTION
**Pull request by Hardikaz**
## Related Issue

Issue number #714

## Description
I'm in GSSoC'23
Added a project which is made using GitHub Users API

### CODE REVIEW

File: src/DB/projects.json [hunk 1]
```diff
@@ -1174,4 +1174,28 @@
...
+    "Instagram": "",
+    "YouTube": "",
+    "Twitter": "https://twitter.com/Hardik_Sharma4"
+  },
+  "Projects": [
+    {
+      "link": "https://hardikaz.github.io/GitHubUsersAPI/",
+      "title": "GitHub Users API",
+      "description": "This project is made by using GitHub Users API and simple HTML, CSS, JavaScript",
+      "tech": "[HTML,CSS,JavaScript]"
```
Comment (priyankarpal) [inline src/DB/projects.json:1191]: also, change these

``` suggestion
      "tech": ["HTML","CSS","JavaScript"]
```

File: src/DB/projects.json [hunk 2]
```diff
@@ -1174,4 +1174,28 @@
...
+    },
+    {
+      "link": "https://hardikaz.github.io/GitHubUsersAPI/",
+      "title": "GitHub Users API",
+      "description": "This project is made by using GitHub Users API and simple HTML, CSS, JavaScript",
+      "tech": "[HTML,CSS,JavaScript]"
```
Comment (priyankarpal) [inline src/DB/projects.json:1197]: same here also

``` suggestion
      "tech": ["HTML","CSS","JavaScript"]
```

**Comment by priyankarpal** [comment]
please add your issue number & if you're in GSSoC'23 please mention so that I can add labels

**Comment by Hardikaz** [comment]
After changing should I create a new issue or just give another pull request?

**Comment by priyankarpal** [comment]
> After changing should I create a new issue or just give another pull request?

just add issue number to the description

**Comment by Hardikaz** [comment]
OK

**Comment by priyankarpal** [comment]
> is it OK now?

there is a conflict error also, I requested changes so please check those.

**Comment by Hardikaz** [comment]
Yeah Pls check I have committed the suggestions.

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**